lib/watir-webdriver/locators/element_locator.rb

```diff
@@ -233,6 +242,21 @@ def delete_regexps_from(selector)
       rx_selector
     end
 
+    def regexp_selector_to_predicates(key, re)
+      return [] if re.casefold?
```

Comment:
I'm not exactly sure, but I believe case-insensitivity can be achieved by using [translate()](https://github.com/watir/watir-webdriver/blob/master/lib/watir-webdriver/xpath_support.rb#L15-17].